Riot/Modules/Room/RoomInfo/RoomInfoCoordinator.swift

```diff
@@ -152,8 +152,13 @@ final class RoomInfoCoordinator: NSObject, RoomInfoCoordinatorType {
                     self.navigationRouter.push(search, animated: animated, popCompletion: nil)
                 }
             })
+        case .notifications:
+            break
         default:
-            segmentedViewController.selectedIndex = target.tabIndex
+            guard let tabIndex = target.tabIndex else {
+                fatalError("No settings tab index for this target.")
```

Comment:
@ismailgulek Thought it might be a little clearer to make tab index optional and call out the error explicitly here rather than assign non existing tab id's for cases that don't navigate to a settings tab. Do you think this makes sense?